README.md

```diff
@@ -591,6 +591,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
 ## Unofficial
  * [Crystal Programming Discord Server](https://discord.gg/YPGkQ5A) - Unofficial Discord server dedicated to the Crystal Programming Language
  * [Russian-speaking Telegram Group](https://t.me/crystal_ru) - Добро пожаловать, товарищ!
+ * [Chinese-speaking Telegram Group](https://t.me/crystal_cn) - 来吧！TG 中文圈的朋友们！
```

Comment:
Please re-range the order to fix `crystal spec`

Reply:
Already fixed